Sources/SwiftSyntax/generated/SyntaxEnum.swift

```diff
@@ -16,8 +16,6 @@
 @frozen // FIXME: Not actually stable, works around a miscompile
 public enum SyntaxEnum {
   case token(TokenSyntax)
-  case accessPathComponent(AccessPathComponentSyntax)
-  case accessPath(AccessPathSyntax)
```

Comment:
Note to myself (no action required on your side) since this is the first time we are trying to do a rename while maintaining source compatibility as much as possible and I’m trying to assess where source breakages might occur. 

We can’t rename a type without also renaming the case in `SyntaxEnum`. I think that it might be fine because there are way fewer people switching over a `SyntaxEnum` than there are using the type names.